docs/platform/access-management/sso-providers/azure-entra-id.md

```diff
@@ -62,15 +62,13 @@ Create client credentials so Airbyte can talk to your application.
 
 #### Configure SSO in Airbyte
 
-:::info
-Currently, this portion of the setup can only be done by an Airbyte employee. Contact Support to proceed.
-:::
-
-1. In Airbyte, click **Settings**.
+1. In Airbyte, click **Organization settings** > **General**.
 
-2. Under **Organization**, click **General**.
+    :::info
+    Currently, this portion of the setup requires an Airbyte employee. Contact Support to proceed.
+    :::
```

Comment:
Is this correct? Don't the following steps describe a self-serve SSO configuration?